modules/nf-core/modules/samtools/collatefastq/main.nf

```diff
@@ -24,12 +24,16 @@ process SAMTOOLS_FASTQ {
     def args2 = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    samtools collate -O -@$task.cpus $args $input . |
+    samtools collate \\
```

Comment:
sorry I know this should have maybe come to mind when you did the upstream changes. How is this effecting people taht don't want to use collate? I mena it is recommended generally, but I suppose there might be reasons people don't want to use it. Would it maybe be an option to handle it similarly to how we do it in the mapping modules with yes/no to sorting?

Reply:
For now it's a different module than the `fastq` one, I assume people that use the `collatefastq` module want to use `collate`.
I was actually wondering if it could be replace by the `bam2fastq` one actually, output seems similar